handbook/engineering/distribution/managed/upgrade_process.md

```diff
@@ -1,8 +1,7 @@
 # Upgrading a managed instance
 
-Managed instances configuration is tracked in [`deploy-sourcegraph-managed-instances`](https://github.com/sourcegraph/deploy-sourcegraph-managed). For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).
-
 - [Walkthrough video](#walkthrough-video)
```

Comment:
```suggestion
Managed instances configuration is tracked in [`deploy-sourcegraph-managed-instances`](https://github.com/sourcegraph/deploy-sourcegraph-managed). For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).

- [Walkthrough video](#walkthrough-video)
```

I think it is worth keeping a brief description of the document and that you might want to look elsewhere for other things in almost all documents we have.